app/internal/app/dashboard_store.go

```diff
@@ -55,24 +64,65 @@ func (s *dashboardStore) GetAll(ctx context.Context) (donor, attorney, certifica
 		return nil, nil, nil, nil
 	}
 
-	var items []*page.Lpa
-	if err := s.dynamoClient.GetAllByKeys(ctx, searchKeys, &items); err != nil {
+	var lpasOrProvidedDetails []interface{}
+	if err := s.dynamoClient.GetAllByKeys(ctx, searchKeys, &lpasOrProvidedDetails); err != nil {
```

Comment:
It will be better, now this is returning various types, to change the signature to `GetAllByKeys(context.Context, keys []Keys) ([]map[string]types.AttributeValue, error)`. Then you'll be able to inspect each list item for `PK` and `SK` to decide what it is, then use `attributevalue.UnmarshalMap` to get the full record

Reply:
This is what I was trying to do in various iterations and failing - makes sense!